ocs_ci/deployment/deployment.py

```
@@ -3747,6 +3748,363 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+            managed_clusters = get_non_acm_cluster_config()
+            if not managed_clusters:
+                logger.warning("[ramenctl-e2e] No managed clusters found")
+
+            for idx, cluster in enumerate(managed_clusters, start=1):
+                cluster_kubeconfig = cluster.RUN.get("kubeconfig")
+                if cluster_kubeconfig and os.path.exists(cluster_kubeconfig):
+                    cluster_dest = os.path.join(ramenctl_dir, f"c{idx}.yaml")
+                    shutil.copy(cluster_kubeconfig, cluster_dest)
+                    clusters_config[f"c{idx}"] = {"kubeconfig": f"c{idx}.yaml"}
+                    logger.debug(
+                        f"[ramenctl-e2e] Copied c{idx} kubeconfig to {cluster_dest}"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)